lib/storage.js

```diff
@@ -46,6 +46,17 @@ Store.prototype._createAppRef = function (id, fbRef, data) {
 
     function newBlock(blockId) {
         var block = blocks[blockId];
+        var localBlock = block;
+        if(block.type === "text") {
+            localBlock.attributes.innerHTML.value = i18n.get(localBlock.attributes.innerHTML.value) || localBlock.attributes.innerHTML.value;
+        } else if(block.type === "phone") {
+            localBlock.attributes.innerHTML.value = i18n.get(localBlock.attributes.innerHTML.value) || localBlock.attributes.innerHTML.value;
+        } else if(block.type === "sms") {
+            localBlock.attributes.innerHTML.value = i18n.get(localBlock.attributes.innerHTML.value) || localBlock.attributes.innerHTML.value;
+        } else if(block.type === "submit") {
+            localBlock.attributes.innerHTML.value = i18n.get(localBlock.attributes.innerHTML.value) || localBlock.attributes.innerHTML.value;
+        }
```

Comment:
```js
if (['text', 'phone', 'sms', 'submit'].indexOf(block.type) !== -1) {
    clone.attributes.innerHTML.value =  i18n.get(block.attributes.innerHTML.value);
}
```
you don't need the `||` since it is taken care of in i18n.js https://github.com/mozilla/webmaker-app/blob/master/lib/i18n.js#L65